store/src/app/shared/components/page-not-found/page-not-found.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="container">
+  <h1 class="title">404</h1>
+  <p>Something went wrong...</p>
+  <app-button routerLink="/products" [buttonName]="buttonName"></app-button>
```

Comment:
Could be just

`buttonName="Back to home"`

Reply:
done